content/docs/05.developer-guide/03.variables/01.index.md

```diff
@@ -16,6 +16,7 @@ The following table lists all the default variables available on each execution.
 | ---------- |-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 |  `{{ flow.id }}` | The identifier of the flow.                                                                                                                                                             |
 |  `{{ flow.namespace }}` | The name of the flow namespace.                                                                                                                                                         |
+|  `{{ flow.tenantId }}` | The identifier of the flow tenant (EE only).                                                                                                                                                         |
```

Comment:
```suggestion
|  `{{ flow.tenantId }}` | The identifier of the tenant (EE only).                                                                                                                                                         |
```

Shorter version like this would be even better: The tenant ID (EE only).